frontend/integration-tests/views/crud.view.ts

```diff
@@ -167,3 +167,25 @@ export const emptyState = $('.cos-status-box').$('.text-center');
 
 export const errorMessage = $('.pf-c-alert.pf-m-inline.pf-m-danger');
 export const successMessage = $('.pf-c-alert.pf-m-inline.pf-m-success');
+
+export const clickYamlCreation = async function() {
+  const createDropdownIsPresent = await createItemButton.isPresent();
+  if (createDropdownIsPresent) {
+    await createItemButton.click();
+    await createYAMLLink.click();
+  } else {
+    await browser.wait(until.presenceOf(createYAMLButton));
+    await createYAMLButton.click();
+  }
+  browser.wait(until.and(untilNoLoadersPresent, until.presenceOf(element(by.cssContainingText('h1', 'Create')))));
+};
+
+export const createNamespacedResourceWithDefaultYaml = async function(resourceType:string) {
```

Comment:
If this is only used in storage scenario, just inline it there.

Reply:
This function could be reused by other scenario if they want to create a resource with default yaml.  So it may be better to put it here, WDYT?